config/webpack/webpack.config.js

```diff
@@ -31,14 +31,20 @@ const {
 
 // port is only required for dev builds
 const makeWebpackConfig = ({
+  isStoryBook = false,
   isIntegration = false,
   port = 0,
   isDevServer = false,
 } = {}) => {
   const { isProductionBuild } = utils;
   const webpackMode = isProductionBuild ? 'production' : 'development';
-
-  const renderHtml = isLibrary ? isDevServer : !isIntegration;
+  const shouldRenderHtml = () => {
+    if (isStoryBook) {
```

Comment:
This can just check `isIntegration`. For context, `isIntegration` means it's an external build system that needs to merge our webpack config (e.g. `storybook` and `playroom`). None of these scenarios should have html rendering setup.

Reply:
Thanks for pointing it out, will make the change, I didn't dare touch any `integration` :)